antiplag/serializers.py

```diff
@@ -24,7 +24,7 @@ def to_representation(self, instance):
 class SubmissionSerializer(serializers.ModelSerializer):
     class Meta:
         model = Submission
-        fields = ("id", "status")
+        fields = ("id", "status", "email")
```

Comment:
Do we need the email field on the frontend side of the app? I don't think so. Please elaborate.

Reply:
Not at this moment but in the future, why it shouldn't be available as a valid field? (E.g. change notification email feature, add it if there was no set etc.)

Reply:
Do we need it now? We rather should not as we don't have any consent from customer. Also, our routes are available using `id` parameter basically to anyone - we don't have authorization yet. Having such personal information in http responses would raise red flag in terms of GDPR compliance.

Reply:
Agreed, removed.